purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/OfflineEntitlementsIntegrationTest.kt

```diff
@@ -0,0 +1,394 @@
+package com.revenuecat.purchases
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.common.sha1
+import com.revenuecat.purchases.factories.StoreProductFactory
+import com.revenuecat.purchases.factories.StoreTransactionFactory
+import com.revenuecat.purchases.helpers.mockQueryProductDetails
+import com.revenuecat.purchases.interfaces.UpdatedCustomerInfoListener
+import com.revenuecat.purchases.models.StoreTransaction
+import io.mockk.every
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import org.junit.After
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.time.Duration
+
+@Suppress("TooManyFunctions")
+@RunWith(AndroidJUnit4::class)
+class OfflineEntitlementsIntegrationTest : BasePurchasesIntegrationTest() {
+
+    private val initialActiveTransaction = StoreTransactionFactory.createStoreTransaction(
+        skus = listOf(Constants.productIdToPurchase),
+        purchaseToken = Constants.googlePurchaseToken
+    )
+    private val initialActivePurchases = mapOf(
+        initialActiveTransaction.purchaseToken.sha1() to initialActiveTransaction
+    )
+
+    @After
+    fun tearDown() {
+        tearDownTest()
```

Comment:
Would it be possible to put `@After` on the parent `tearDownTest` so subclasses don't have to remember this?